cmds/webboot/webboot_test.go

```diff
@@ -183,6 +183,76 @@ func TestBackOption(t *testing.T) {
 	}
 }
 
+func TestDisplayChecksumPrompt(t *testing.T) {
+	// test data
+	var testDistros = map[string]Distro{
+		"FakeDistro": {
+			checksum:     "1234567",
+			checksumType: "sha256",
+		},
+		"FakeDistroNoChecksum": {},
+		"FakeDistroGoodChecksum": {
+			checksum:     "407dc87b95afbe268e760313971041860f36e953a2116db03418a98ce46d61bc",
+			checksumType: "sha256",
+		},
+	}
+
+	type test struct {
+		name       string
+		keyInput   []string
+		distroName string
+		want       string
```

Comment:
I think a better name for this variable would be expectedValue or expectedOutput.

Reply:
I think using `want` is okay. It is used in the Go example here and elsewhere: https://golang.org/doc/tutorial/add-a-test